Sources/Sentry/PrivateSentrySDKOnly.mm

```diff
@@ -320,31 +321,19 @@ + (nullable SentrySessionReplayIntegration *)getReplayIntegration
 
     return replayIntegration;
 }
-#endif
 
 + (void)captureReplay
 {
-#if SENTRY_HAS_UIKIT && !TARGET_OS_VISION
     [[PrivateSentrySDKOnly getReplayIntegration] captureReplay];
-#else
-    SENTRY_LOG_DEBUG(
-        @"SentrySessionReplayIntegration only works with UIKit enabled and target is "
-        @"not visionOS. Ensure you're using the right configuration of Sentry that links UIKit.");
-#endif
```

Comment:
I've removed these because they couldn't get ever triggered anyway - the header doesn't define these functions when replay is not available.